minimal_redux_poc/package.json

```diff
@@ -8,9 +8,12 @@
     "server": "node_modules/.bin/http-server",
     "test": "npm run build && npm run lint && node_modules/.bin/jest",
     "test:watch": "npm test -- --watch",
-    "build": "node_modules/.bin/webpack",
-    "build:watch": "node_modules/.bin/webpack --watch",
-    "start": "npm run build && concurrently \"npm run build:watch\" \"npm run server -- -p 4444\""
+    "build": "NODE_ENV=production webpack",
+    "build:watch": "NODE_ENV=development webpack --watch",
+    "start": "NODE_ENV=development npm run build && concurrently \"npm run build:watch\" \"npm run server -- -p 4444\""
+  },
+  "eslintConfig": {
+    "extends": "react-app"
```

Comment:
How does this work with the `.eslintrc` file which extends `airbnb`. Should that just be an array and specified there?

Reply:
eslint will read both config from `.eslintrc` and the `package.json`.  I agree that it would be cleaner to put that in the `.eslintrc`.

Reply:
Gotcha. Do you mind making that small change here?

Reply:
it turns out that eslint does not read the package.json if there is an `.eslintrc`.  if it had, then `eslint-config-react-app` (as well as `eslint-plugin-flowtype`) would have been required as a dependencies.  this is a useful extension for accessibility checks, so I do not think that there is a downside to adding it here.  note, however, that it is entirely optional.